README.md

```diff
@@ -169,7 +169,7 @@ Use the npm package manager for Node-based projects. USWDS maintains the [`@uswd
 
 1. Install `Node/npm`. Below is a link to find the install method that coincides with your operating system:
 
-   - Node v12.13.2 (current LTS), [Installation guides](https://nodejs.org/en/download/)
+   - Node v12.13.2 (current LTS), [Installation guides](https://nodejs.org/en/download)
```

Comment:
**question:** Current LTS is v22 and our `.nvmrc` lists v20. Should we update the listed version as part of this update?
```suggestion
   - Node v20, [Installation guides](https://nodejs.org/en/download)
```

Reply:
Let's update it to v20 to unblock this. 

Can we update node in #6208 @mahoneycm @cathybaptista?